app/views/new_admin/allocation_chart/_allocations.html.erb

```diff
@@ -28,11 +28,21 @@
           </div>
         </td>
         <td class="px-4 py-3">
-          <div class="flex items-center text-sm">
-            <p class="text-sm text-gray-600 dark:text-gray-400">
-              <%= allocation.end_at %>
-            </p>
-          </div>
+          <% if allocation.id %>
```

Comment:
the link isn't necessary if the allocation doesn't exist

Reply:
If the allocation does not exist, the variable allocation should be nil, right?

Reply:
the allocation variable will still exist with the id as `nil`, but it will have a `user_id`. On the allocation chart it displays `Not Allocated` under the `Ends At` column if the allocation doesn't exist, if the user is allocated it will display the date with a link to the details of the allocation